.ci/docker/examples/arm/setup.sh

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env bash
```

Comment:
I am not sure if we want to move examples dir under docker. This was meant as an example for the end user, it happens to be a good setup for CI.

Reply:
I kind of agree on this point, it looks weird having a bunch of scripts under `.ci/docker/examples/arm`, so I try to think of them as installation scripts to setup arm job.

There are some rules w.r.t. the building of Docker images:

* Only `.ci/docker` folder is taken into account per convention.  This PR tries to follow this rule, but it looks bad in this case.
* The rebuilding of the Docker image happens.  But because its hash version, calculated by `DOCKER_TAG=$(git rev-parse HEAD:"${DOCKER_BUILD_DIR}")`, doesn't change, the image becomes stale.  

I think I'll keep this PR around just in case, and see if I can bend the second rule instead to force update the image.  Note that force update, even if it could be done, can only be an optional choice because not overwriting an existing image helps CI reliability from my experience.